integration_tests/macros/system/log_config_var.sql

```diff
@@ -0,0 +1,6 @@
+{# Loging the wanted config var as an elementary log (using elementary.edr_log) #}
+{# The dbtRunner catch this log when executed with run_operation #}
+{# This is used for accessing the integration tests vars #}
+{% macro log_config_var(var_name) %}
```

Comment:
cosmetics - I'd call it return_config_var (the log mechanism is our way to return it back to the python)